### PR TITLE
Added logos to Service pages

### PR DESCRIPTION
**Pull request by vishakgowda**
Added SVG logos to service pages

### CODE REVIEW

File: app/services/ui-development.html [hunk 1]
```diff
@@ -100,6 +100,11 @@ <h2 class="title my-4">Front-end development</h2>
         <p>
           We have extensive experience in working with:
         </p>
+        <picture class="d-flex justify-content-between mt-5">
```
Comment (varunnotkar59) [inline app/services/ui-development.html]: Please correct this. One picture tag can have only one image. We add multiple image tags only when we are serving images for different screen sizes.

Ex: 
`<picture>

</picture>`

File: app/services/web-development.html [hunk 2]
```diff
@@ -105,6 +105,17 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+        <picture class="d-flex justify-content-around">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Please correct this. One picture tag can have only one image. We add multiple image tags only when we are serving images for different screen sizes.

Ex:
`<picture>  </picture>`

File: app/services/web-development.html [hunk 3]
```diff
@@ -105,6 +105,17 @@ <h2 class="title my-4">Websites</h2>
...
+        <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
+        <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
+        <img src="../images/services/wordpress-logo.svg" alt="wordpress logo" title="Wordpress" class="img-fluid">
+        <img src="../images/services/gulp-logo.svg" alt="gulp logo" title="Gulp" class="img-fluid">
+      </picture>
+      <picture class="d-flex justify-content-around mt-4">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Please correct this. One picture tag can have only one image. We add multiple image tags only when we are serving images for different screen sizes.

Ex:
`<picture>  </picture>`

File: app/services/web-development.html [hunk 4]
```diff
@@ -105,6 +105,17 @@ <h2 class="title my-4">Websites</h2>
...
+          <img src="../images/services/bower-logo.svg" alt="bower logo" title="Bower" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Fix code alignment

File: app/services/custom-applications.html [hunk 5]
```diff
@@ -103,9 +103,19 @@ <h2 class="title my-4">Product Development</h2>
...
+        </picture>
+        <picture class="d-flex justify-content-between flex-wrap mt-lg-4 ">
+            <img src="../images/services/amazon-web-services-logo.svg" alt="aws logo" title="aws" class="img-fluid">
+            <img src="../images/services/mysql-logo.svg" alt="MySQL logo" title="MySQL" class="img-fluid">
+            <img src="../images/services/react-logo.svg" alt="React logo" title="React" class="img-fluid">
+          </picture>
       </div>
     </div>
-  </section>
+  </sec ion>
```
Comment (varunnotkar59) [inline app/services/custom-applications.html]: Syntax error

**Review by varunnotkar59 — CHANGES_REQUESTED** [review]
Please make the requested changes

File: app/services/ui-development.html (hunk 1 above)
Comment (vishakgowda) [reply app/services/ui-development.html]: Picture tag added to all images in service pages.

File: app/services/web-development.html (hunk 2 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Picture tag added to all images in service pages

File: app/services/web-development.html (hunk 3 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Picture tag added to all images in service pages.

File: app/services/web-development.html (hunk 4 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Code allignement is fixed

File: app/services/custom-applications.html (hunk 5 above)
Comment (vishakgowda) [reply app/services/custom-applications.html]: Fixed the syntax error

File: app/services/custom-applications.html [hunk 6]
```diff
@@ -103,6 +103,28 @@ <h2 class="title my-4">Product Development</h2>
         <p class="highlight">
           We have extensive experience in working with:
         </p>
+
+        <div class="d-flex justify-content-between align-items-center flex-wrap">
+          <picture>
+            <img src="../images/services/angular-logo.svg" alt="Angular logo" title="Angular" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/laravel-logo.svg" alt="laravel logo" title="laravel" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/custom-applications.html]: Capitalization in `title` attribute

File: app/services/custom-applications.html [hunk 7]
```diff
@@ -103,6 +103,28 @@ <h2 class="title my-4">Product Development</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/django-logo.svg" alt="Django logo" title="django" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/custom-applications.html]: Capitalization in `title` attribute

File: app/services/custom-applications.html [hunk 8]
```diff
@@ -103,6 +103,28 @@ <h2 class="title my-4">Product Development</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/amazon-web-services-logo.svg" alt="aws logo" title="aws"
```
Comment (varunnotkar59) [inline app/services/custom-applications.html]: Capitalization in `alt` and `title` attributes. This one should be *AWS*

File: app/services/custom-applications.html [hunk 9]
```diff
@@ -103,6 +103,28 @@ <h2 class="title my-4">Product Development</h2>
...
+              class="img-fluid"></picture>
+          <picture>
+            <img src="../images/services/mysql-logo.svg" alt="MySQL logo" title="MySQL" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/react-logo.svg" alt="React logo" title="React" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/custom-applications.html]: Please make this **React.js**

File: app/services/ui-development.html [hunk 10]
```diff
@@ -100,6 +100,19 @@ <h2 class="title my-4">Front-end development</h2>
         <p>
           We have extensive experience in working with:
         </p>
+
+        <div class="d-flex justify-content-between align-items-center mt-5">
+          <picture>
+            <img src="../images/services/react-logo.svg" alt="React Logo" title="React" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/ui-development.html]: Please make this **React.js**

Comment (varunnotkar59) [reply app/services/ui-development.html]: in the `alt` attribute please change *Logo* to **logo**

File: app/services/ui-development.html [hunk 11]
```diff
@@ -100,6 +100,19 @@ <h2 class="title my-4">Front-end development</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/angular-logo.svg" alt="Angular Logo" class="img-fluid" title="Angular">
```
Comment (varunnotkar59) [inline app/services/ui-development.html]: in the `alt` attribute please change *Logo* to **logo**

File: app/services/ui-development.html [hunk 12]
```diff
@@ -100,6 +100,19 @@ <h2 class="title my-4">Front-end development</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/vue-logo.svg" alt="Vue logo" title="Vue" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/ui-development.html]: Please make this **Vue.js**

File: app/services/web-development.html [hunk 13]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+
+        <div class="d-flex justify-content-between align-items-center flex-wrap">
+          <picture>
+            <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt`

File: app/services/web-development.html [hunk 14]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt`

File: app/services/web-development.html [hunk 15]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/wordpress-logo.svg" alt="wordpress logo" title="Wordpress" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct spelling to **WordPress** and capitalization in `alt` and `title`.

File: app/services/web-development.html [hunk 16]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/gulp-logo.svg" alt="gulp logo" title="Gulp" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt`

File: app/services/web-development.html [hunk 17]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+        </div>
+        <div class="d-flex justify-content-between align-items-center flex-wrap mt-4">
+          <picture>
+            <img src="../images/services/bower-logo.svg" alt="bower logo" title="Bower" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt`

File: app/services/web-development.html [hunk 18]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/gatsbyjs-logo.svg" alt="gatsby logo" title="Gatsby" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt`

File: app/services/web-development.html [hunk 19]
```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
...
+          </picture>
+          <picture>
+            <img src="../images/services/grunt-logo.svg" alt="grunt logo" title="GRUNT" class="img-fluid">
```
Comment (varunnotkar59) [inline app/services/web-development.html]: Correct capitalization in `alt` and `title`

**Review by varunnotkar59 — CHANGES_REQUESTED** [review]
Fix typo errors

File: app/services/custom-applications.html (hunk 6 above)
Comment (vishakgowda) [reply app/services/custom-applications.html]: All Alt & Title attribute are capitalized

File: app/services/custom-applications.html (hunk 7 above)
Comment (vishakgowda) [reply app/services/custom-applications.html]: All Alt & Title attribute are capitalized

File: app/services/custom-applications.html (hunk 8 above)
Comment (vishakgowda) [reply app/services/custom-applications.html]: All Alt & Title attribute are capitalized

File: app/services/custom-applications.html (hunk 9 above)
Comment (vishakgowda) [reply app/services/custom-applications.html]: Fixed this typo in alt & title attribute

File: app/services/ui-development.html (hunk 10 above)
Comment (vishakgowda) [reply app/services/ui-development.html]: Typo error is fixed

File: app/services/ui-development.html (hunk 11 above)
Comment (vishakgowda) [reply app/services/ui-development.html]: Typo error is fixed

File: app/services/ui-development.html (hunk 12 above)
Comment (vishakgowda) [reply app/services/ui-development.html]: Typo error is fixed

Comment (vishakgowda) [reply app/services/ui-development.html]: Fixed this typo in alt & title attribute

File: app/services/web-development.html (hunk 13 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Typo error is fixed

File: app/services/web-development.html (hunk 14 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Fixed all typo errors & all pages are validated in html validator

File: app/services/web-development.html (hunk 15 above)
Comment (vishakgowda) [reply app/services/web-development.html]: WordPress typo error is fixed

File: app/services/web-development.html (hunk 16 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Fixed this typo in alt & title attribute

File: app/services/web-development.html (hunk 17 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Fixed this typo in alt & title attribute

File: app/services/web-development.html (hunk 18 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Fixed this typo in alt & title attribute

File: app/services/web-development.html (hunk 19 above)
Comment (vishakgowda) [reply app/services/web-development.html]: Fixed this typo in alt & title attribute

**Review by varunnotkar59 — APPROVED** [review]
All review comments implemented. Approving the changes.

**Pull request merged**